sources/packages/web/src/types/formio/custom-components/components/number-currency.ts

```diff
@@ -0,0 +1,27 @@
+export const CURRENCY = {
```

Comment:
Instead of calling the directory as `components` which is very generic, I would suggest inputs.

![image](https://github.com/user-attachments/assets/afd6ec96-3ac9-4494-a50a-4b452c874238)